client/src/main/scala/com/dwolla/cloudflare/domain/model/ratelimits/package.scala

```diff
@@ -0,0 +1,173 @@
+package com.dwolla.cloudflare.domain.model
+
+import com.dwolla.circe._
+import io.circe.{Decoder, Encoder}
+import shapeless.tag.@@
+
+package object ratelimits {
+
+  type RateLimitId = String @@ RateLimitIdTag
+
+  private[cloudflare] val tagRateLimitId: String ⇒ RateLimitId = shapeless.tag[RateLimitIdTag][String]
+
+}
+
+package ratelimits {
+  import java.time.Duration
+
+  import io.circe.ObjectEncoder
+  import io.circe.export.Exported
+  import io.circe.generic.auto._
+
+  trait RateLimitIdTag
+
+  case class RateLimit(id: Option[RateLimitId] = None,
```

Comment:
If I'm interpreting the diff correctly, it looks like `CreateRateLimit` (without an ID) was removed and `RateLimit` was changed to make the ID optional. Are there circumstances when an existing rate limit does not have an ID?

Reply:
It's specified as optional in the documentation: 

![Screen Shot 2019-09-20 at 12 07 05 PM](https://user-images.githubusercontent.com/597530/65345183-4545dc00-db9f-11e9-9893-59ad4773e28e.png)